### PR TITLE
Bump github.com/tektoncd/pipeline from 0.48.0 to 0.49.0 

### PR DESCRIPTION
**Pull request by piyush-garg**
# Changes

Bump github.com/tektoncd/pipeline from 0.48.0 to 0.49.0 

This will fix the test as the default-api-field value is changed to
beta in 0.49 pipeline

Also fix the triggers test by using the value from triggers pkg
instead of pipeline pkg

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Run `make test lint` before submitting a PR
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
Bump github.com/tektoncd/pipeline from 0.48.0 to 0.49.0 
```

### CODE REVIEW

File: pkg/apis/operator/v1alpha1/tektonpipeline_defaults_test.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func Test_SetDefaults_PipelineProperties(t *testing.T) {
 		RequireGitSshSecretKnownHosts:            ptr.Bool(false),
 		EnableTektonOciBundles:                   ptr.Bool(false),
 		EnableCustomTasks:                        ptr.Bool(true),
-		EnableApiFields:                          "stable",
+		EnableApiFields:                          "beta",
```
Comment (jkandasa) [inline pkg/apis/operator/v1alpha1/tektonpipeline_defaults_test.go:50]: @piyush-garg can you update the test values from `github.com/tektoncd/pipeline/pkg/apis/config` => `config.DefaultEnableAPIFields`

Comment (piyush-garg) [reply pkg/apis/operator/v1alpha1/tektonpipeline_defaults_test.go:50]: @jkandasa in the tests if you see we specifically use the values by specifying, so we know about the changes and can take action accordingly.

**Review by vdemeester — APPROVED** [review]

File: pkg/apis/operator/v1alpha1/tektonpipeline_defaults_test.go (hunk 1 above)
Comment (jkandasa) [reply pkg/apis/operator/v1alpha1/tektonpipeline_defaults_test.go:50]: @piyush-garg ok, I thought, in the future if there is a change in `default-api-field `, no need to touch this code base.

**Pull request merged**